components/brave_wallet_ui/page/screens/onboarding/restore-from-recovery-phrase/restore-from-extension.tsx

```diff
@@ -186,14 +162,9 @@ export const OnboardingRestoreFromExtension = ({ restoreFrom }: Props) => {
     switch (currentStep) {
       case 'currentPassword':
         return {
-          title:
-            restoreFrom === 'metamask'
-              ? getLocale('braveWalletMetaMaskExtensionDetected')
-              : getLocale('braveWalletCryptoWalletsDetected'),
+          title: getLocale('braveWalletMetaMaskExtensionDetected'),
           description:
-            restoreFrom === 'metamask'
-              ? getLocale('braveWalletMetaMaskExtensionImportDescription')
-              : getLocale('braveWalletImportBraveLegacyDescription')
+              getLocale('braveWalletMetaMaskExtensionImportDescription')
```

Comment:
Also need to remove `restoreFrom` from list of deps for this function to pass lint